### PR TITLE
Website docs 

### LINKED ISSUE #71: Add website docs
- [x] I've checked that this isn't a duplicate issue.

We should probably make a `README.md` or `CONTRIBUTING.md` file in the `site` folder with instructions on how to clone the website, start the gulp webserver, ect.

### PR DESCRIPTION
**Pull request by Jacob8765**
Adds a `README.md` file with some basic instructions on how to start the webserver. Fixes #71

### CODE REVIEW

File: site/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# About
+Source code for the https://devswag.io website.
+
+# Running locally
+1. Clone this repository by running `git clone https://github.com/swapagarwal/swag-for-dev.git`
```
Comment (aslafy-z) [inline site/README.md]: I'd rater use codeblocks for commands, easier to copy/paste, but that's personal

File: site/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+2. Ensure you are running an up-to-date version of [Node.js](https://nodejs.org/en/) on your machine, and that you have npm installed.
+3. Make sure you have [Gulp](https://gulpjs.com/) installed by running `npm install gulp -g`
+4. Open a terminal in the `swag-for-dev/site` directory. Run `npm install` to install the dependencies, and then `gulp default` to start the webserver
```
Comment (aslafy-z) [inline site/README.md]: `gulp default` could be simplified to `gulp`

File: site/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+5. The app will open itself in a browser after it compiles
```
Comment (aslafy-z) [inline site/README.md]: Provide a link: http://localhost:8000

File: site/README.md (hunk 1 above)
Comment (Jacob8765) [reply site/README.md]: I added a code block for the `clone` command. Should I put the other commands in blocks also?

File: site/README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# About
+Source code for the https://devswag.io website.
+
+# Running locally
+1. Clone this repository by running 
+```
+git clone https://github.com/swapagarwal/swag-for-dev.git
+```
+2. Ensure you are running an up-to-date version of [Node.js](https://nodejs.org/en/) on your machine, and that you have npm installed.
+3. Make sure you have [Gulp](https://gulpjs.com/) installed by running `npm install gulp -g`
```
Comment (aslafy-z) [inline site/README.md]: Please no ! They have to install globaly `gulp-cli`, not `gulp` which is already provided by the local package.json.

Comment (Jacob8765) [reply site/README.md]: Whoops, my bad. Should I change it to `npm install gulp-cli -g`, or just omit it since it's already in package.json?

Comment (aslafy-z) [reply site/README.md]: The CLI is required to use Gulp, so yeah, they need to install `gulp-cli` globally as you wrote.

Comment (Jacob8765) [reply site/README.md]: Alright, I'll change it to that. Thanks!

File: site/README.md (hunk 1 above)
Comment (aslafy-z) [reply site/README.md]: I'd do it for the npm install too, it's up to you

File: site/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+## About
+Source code for the https://devswag.io website.
+
+## Development
+1. Clone this repository by running 
+```
+git clone https://github.com/swapagarwal/swag-for-dev.git
+```
+2. Ensure you are running an up-to-date version of [Node.js](https://nodejs.org/en/) on your machine, and that you have npm installed.
```
Comment (aslafy-z) [inline site/README.md]: Use this url instead : https://nodejs.org/en/download/package-manager/

File: site/README.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+3. Make sure you have [Gulp](https://gulpjs.com/) installed by running
```
Comment (aslafy-z) [inline site/README.md]: Use this url instead: https://gulpjs.org/getting-started.html

**Comment by plibither8** [comment]
@swapagarwal @aslafy-z Lets merge this?

**Comment by hugovk** [comment]
How about adding a link from the main README to the site/README?

https://github.com/swapagarwal/swag-for-dev#contributing

**Comment by swapagarwal** [comment]
@plibither8 Yep, this is almost done.
@Jacob8765 Can you add commands for getting `node / npm` as well?

File: site/README.md [hunk 7]
```diff
@@ -1,24 +1,26 @@
 # devswag.io
 
 ## About
-Source code for the https://devswag.io website.
+
+This folder contains the source code of the devSwag website (hosted at http://devswag.io).
```
Comment (hugovk) [inline site/README.md]: Use HTTPS

Comment (hugovk) [reply site/README.md]: How about renewing the certificate when it expires? :)

Comment (swapagarwal) [reply site/README.md]: I agree. 😂 
It's mostly a matter of more control (one-way door vs two-way doors). If you have strong reason against it, please share and we can discuss that. 💯

Comment (aslafy-z) [reply site/README.md]: Netlify is configured to autorenew the certificate, right ? If yes, I recommend we force https as the cert will always be valid.

Comment (Ir1d) [reply site/README.md]: you might need acme.sh

Comment (aslafy-z) [reply site/README.md]: Please dont, netlify offers it integration: https://www.netlify.com/docs/ssl/#netlify-certificates

**Comment by aslafy-z** [comment]
@swapagarwal NPM is distributed with NodeJS nowadays
There's a link pointing to NodeJS's official website already included in the readme who explains how to install it with various OSs/Packages manager: https://nodejs.org/en/download/package-manager/.

**Comment by swapagarwal** [comment]
Thanks @Jacob8765! 👍

**Pull request merged**